bindings/Modules/src/SofaPython3/SofaDeformable/CMakeLists.txt

```diff
@@ -27,6 +26,6 @@ SP3_add_python_module(
     DESTINATION  Sofa
     SOURCES      ${SOURCE_FILES}
     HEADERS      ${HEADER_FILES}
-    DEPENDS      SofaBase SofaDeformable SofaPython3::Plugin SofaPython3::Bindings.Sofa.Core
+    DEPENDS      Sofa.Component.SolidMechanics.Spring SofaPython3::Plugin SofaPython3::Bindings.Sofa.Core
```

Comment:
Should be propagated to [Bindings.ModulesConfig.cmake.in](https://github.com/sofa-framework/SofaPython3/blob/master/bindings/Modules/Bindings.ModulesConfig.cmake.in)